config/ldes-delta-pusher/publisher.ts

```diff
@@ -29,9 +29,8 @@ const fetchSubjectData = async (
       ${properties.map((p) => sparqlEscapeUri(p)).join("\n")}
     }`;
   }
-
   const filter =
-    typeof subject.ldesType === "object" ? subject.ldesType[target].filter : "";
+    typeof subject.ldesType === "object" && subject.ldesType[target].filter ? subject.ldesType[target].filter : "";
```

Comment:
Added this because the service was exited with code 1 because in `fetchSubjectData()` the filter value was set to undefined in the executed query

```bash
ldes-delta-pusher-1  | 	 SUBJECT: {
ldes-delta-pusher-1  |   public: {
ldes-delta-pusher-1  |     filter: 'FILTER(?p NOT IN (<http://data.vlaanderen.be/ns/persoon#heeftGeboorte>, <http://www.w3.org/ns/adms#identifier>, <http://data.vlaanderen.be/ns/persoon#geslacht>))'
ldes-delta-pusher-1  |   },
ldes-delta-pusher-1  |   abb: {},
ldes-delta-pusher-1  |   internal: {}
ldes-delta-pusher-1  | }
ldes-delta-pusher-1  | 	 TARGET: internal
ldes-delta-pusher-1  |
ldes-delta-pusher-1  |     PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
ldes-delta-pusher-1  |     CONSTRUCT {
ldes-delta-pusher-1  |       <http://data.lblod.info/id/personen/f202051a-f5c0-4aa4-8b6b-7980a8239ed5> ?p ?o .
ldes-delta-pusher-1  |       <http://data.lblod.info/id/personen/f202051a-f5c0-4aa4-8b6b-7980a8239ed5> ext:relatedTo ?bestuurseenheid .
ldes-delta-pusher-1  |     } WHERE {
ldes-delta-pusher-1  |       GRAPH ?g {
ldes-delta-pusher-1  |         <http://data.lblod.info/id/personen/f202051a-f5c0-4aa4-8b6b-7980a8239ed5> ?p ?o .
ldes-delta-pusher-1  |         ?org a <http://data.vlaanderen.be/ns/besluit#Bestuursorgaan>.
ldes-delta-pusher-1  |       }
ldes-delta-pusher-1  |       ?g ext:ownedBy ?bestuurseenheid .
ldes-delta-pusher-1  |
ldes-delta-pusher-1  |       undefined
ldes-delta-pusher-1  |       FILTER NOT EXISTS {
ldes-delta-pusher-1  |         ?g a <http://mu.semte.ch/vocabularies/ext/FormHistory> .
ldes-delta-pusher-1  |       }
ldes-delta-pusher-1  |     }
```